src/icons/scripts/svgo-config.ts

```diff
@@ -0,0 +1,53 @@
+import { type Config } from 'svgo';
+
+export const configuration: Config = {
+  plugins: [
+    'cleanupAttrs',
+    'removeDoctype',
+    'removeXMLProcInst',
+    'removeComments',
+    'removeMetadata',
+    'removeTitle',
+    'removeDesc',
+    'removeUselessDefs',
+    'removeEditorsNSData',
+    'removeEmptyAttrs',
+    'removeHiddenElems',
+    'removeEmptyText',
+    'removeEmptyContainers',
+    'cleanupEnableBackground',
+    'convertStyleToAttrs',
+    'convertStyleToAttrs',
+    'convertColors',
+    'convertPathData',
+    'convertTransform',
+    'removeUnknownsAndDefaults',
+    'removeNonInheritableGroupAttrs',
+    'removeUnusedNS',
+    'cleanupIds',
+    'cleanupNumericValues',
+    'moveElemsAttrsToGroup',
+    'moveGroupAttrsToElems',
+    'collapseGroups',
+    'removeRasterImages',
+    'mergePaths',
+    'convertShapeToPath',
+    'sortAttrs',
+    'removeDimensions',
+    'removeXMLNS',
+    {
+      name: 'convertColors',
+      params: {
+        currentColor: true,
+      },
+    },
+    {
+      name: 'preset-default',
+      params: {
+        overrides: {
+          removeViewBox: false,
+        },
+      },
+    },
+  ],
+};
```

Comment:
let's use satisfies here

```suggestion
export const configuration = {
  plugins: [
    'cleanupAttrs',
    'removeDoctype',
    'removeXMLProcInst',
    'removeComments',
    'removeMetadata',
    'removeTitle',
    'removeDesc',
    'removeUselessDefs',
    'removeEditorsNSData',
    'removeEmptyAttrs',
    'removeHiddenElems',
    'removeEmptyText',
    'removeEmptyContainers',
    'cleanupEnableBackground',
    'convertStyleToAttrs',
    'convertStyleToAttrs',
    'convertColors',
    'convertPathData',
    'convertTransform',
    'removeUnknownsAndDefaults',
    'removeNonInheritableGroupAttrs',
    'removeUnusedNS',
    'cleanupIds',
    'cleanupNumericValues',
    'moveElemsAttrsToGroup',
    'moveGroupAttrsToElems',
    'collapseGroups',
    'removeRasterImages',
    'mergePaths',
    'convertShapeToPath',
    'sortAttrs',
    'removeDimensions',
    'removeXMLNS',
    {
      name: 'convertColors',
      params: {
        currentColor: true,
      },
    },
    {
      name: 'preset-default',
      params: {
        overrides: {
          removeViewBox: false,
        },
      },
    },
  ],
} satisfies Config;
```

Reply:
@cfernando415 The use of satisfies Config might be unnecessary in this context. We're not relying on type inference from configuration, we just need type type safety and Intellisense. 

P.S I'm not opposed to this change, I just think that reaching out for `satisfies` is mostly beneficial when we need both type safety and more specific inference. I'd like to hear your thoughts

Reply:
hmmm... that is a valid point in this context and no other properties are being used that would require type safety as you mentioned.

I am good with that reasoning - no change needed for this

resolving this thread